<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerVWidget</class>
 <widget class="QWidget" name="PlayerVWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>208</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="albumCover">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>95</width>
           <height>95</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../icons/logo.png</normaloff>../icons/logo.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MDataWidget" name="mdata_w" native="true"/>
       </item>
       <item>
        <widget class="VolumeWidget" name="volumectl_w" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="bottomHLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="PlayCtlWidget" name="playctl_w" native="true"/>
       </item>
       <item>
        <widget class="ProgressWidget" name="progress_w" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VolumeWidget</class>
   <extends>QWidget</extends>
   <header>GUI/volumewidget/volumewidget.h</header>
  </customwidget>
  <customwidget>
   <class>MDataWidget</class>
   <extends>QWidget</extends>
   <header>GUI/mdatawidget/mdatawidget.h</header>
  </customwidget>
  <customwidget>
   <class>PlayCtlWidget</class>
   <extends>QWidget</extends>
   <header>GUI/playctlwidget/playctlwidget.h</header>
  </customwidget>
  <customwidget>
   <class>ProgressWidget</class>
   <extends>QWidget</extends>
   <header>GUI/progresswidget/progresswidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../upplay.qrc"/>
 </resources>
 <connections/>
</ui>
